src/components/Resource/ResourceCreate.tsx

```diff
@@ -183,14 +201,21 @@ export default function ResourceCreate(props: ResourceProps) {
                     render={({ field }) => (
                       <FormItem>
                         <FormLabel required>
-                          {t("facility_for_care_support")}
+                          {t("facility_assign_request")}
```

Comment:
```suggestion
                          {t("facility_for_care_support")}
```

Let's switch this back as it was a product request.